src/test/java/bob/task/TaskListTest.java

```diff
@@ -0,0 +1,39 @@
+package bob.task;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.io.PrintStream;
+import java.io.ByteArrayOutputStream;
+
+public class TaskListTest {
+    private ByteArrayOutputStream outputStreamCaptor = new ByteArrayOutputStream();
+    private TaskList taskList;
+
+    @BeforeEach
+    public void setUp() {
+        System.setOut(new PrintStream(outputStreamCaptor));
+        ArrayList<Task> tasks = new ArrayList<>();
+        tasks.add(new Todo("Code"));
+        tasks.add(new Deadline("Submit quiz", LocalDate.parse("2024-09-10")));
+        tasks.add(new Event("Birthday", "Monday", "5pm", "9pm"));
+        taskList = new TaskList(tasks);
+    }
```

Comment:
Hi. setUp() might not be informative enough as to what the test case is suppose to do. Perhaps you could rename it to 
initialiseTaskListTest().

Reply:
Oh that's a great suggestion, thanks! I have changed it.